### PR TITLE
allow independent padding amount for either end of slider

### PR DESCRIPTION
**Pull request by nlundquist**
Let a user pass the padding as a single number as before, or an array of 2 numbers independently specifying the padding for either end of the slider.

Let me know if there are any changes you need before merging.

### CODE REVIEW

**Comment by leongersen** [comment]
I like this change! I'm not seeing any issues in my (read-only) review of the changes. A couple unit tests and changes to the docs would be necessary, though. After that, @QuentinRoy, would you be willing to release this?

**Comment by nlundquist** [comment]
Thanks for reviewing this. I'll add tests & doc changes this coming Monday.

**Comment by QuentinRoy** [comment]
Sorry, I just saw this. Yes of course, I can deal with the release.

File: distribute/nouislider.js [hunk 1]
```diff
@@ -649,25 +649,33 @@
...
 
-		parsed.padding = parsed.spectrum.getMargin(entry);
+		if ( !Array.isArray(entry) ) {
+			entry = [entry, entry];
+		}
+
+		parsed.padding = [parsed.spectrum.getMargin(entry[0]), parsed.spectrum.getMargin(entry[1])];
 
-		if ( !parsed.padding ) {
+		if ( !parsed.padding[0] || !parsed.padding[1] ) {
```
Comment (leongersen) [inline distribute/nouislider.js]: This probably breaks if padding for one of the two sides is set to `0`.

**Pull request merged**